src/components/Loans/OpenLoans/helpers/getListDataFormatter.js

```diff
@@ -49,7 +49,8 @@ export default function getListDataFormatter(
         }
         return '-';
       },
-      sorter: loan => get(loan, ['item', 'title']),
+      sorter: loan => get({ ...loan, item: { ...loan.item, title: loan.item.title.toUpperCase() } },
+        ['item', 'title'])
```

Comment:
I think you can simplify it to 
`get(loan, ['item', 'title'])?.toLowerCase()`
 or
`get(loan, ['item', 'title'], '').toLowerCase()` 

I am not sure that it is correct approach to use `get` to get `title` if you execute `loan.item.title` inside.

Reply:
Sure, updated.